src/MatBlazor/Components/MatSnackbar/BaseMatSnackbar.cs

```diff
@@ -29,14 +32,41 @@ public bool IsOpen
                 if (IsOpen != value)
                 {
                     _isOpen = value;
-                    CallAfterRender(async () =>
+                    CallAfterRender(async () => await SetIsOpen(value));
+                    if (_isOpen == false)
```

Comment:
@henon I think this is a code smell unless it's a nullable bool (which it isn't in this case)